src/cheri-pte-ext.adoc

```diff
@@ -167,6 +167,13 @@ and the capability read from memory optionally has its tag set^1^.
 
 ^2^ See <<section_hardware_pte_updates>>.
 
+The exceptions added by {cheri_pte_ext_name}:
+
+* reuse the load page fault and store/AMO page fault exception cause values,
+and so the cause of the exception can be determined by software by checking bit zero of <<mtval2>>,
+<<stval2>> etc.
+* are all prioritized _above_ standard page fault exceptions.
```

Comment:
Wait actually does this include the page faults that check the tag on the loaded capability?

> A load page fault exception is raised when a capability load or AMO instruction is executed with [C-permission](https://riscv.github.io/riscv-cheri/#c_perm) granted and the virtual page’s CRG bit does not equal [sstatus](https://riscv.github.io/riscv-cheri/#sstatusreg_pte).CRG **and the capability read from memory optionally has its tag set**.

That must happen after the RWXU checks I think? Should it happen before the A/D page fault too?

In fact is it possible for the A/D bits to be updated and *then* you get a page fault from loading a tagged capability?

Reply:
ok - fixed - please have another look

Reply:
Yeah I agree with your comment. I'm not sure the spec update makes that clear though because it still says

> * are all prioritized _above_ standard page fault exceptions.

Maybe something like:

> * are all prioritized _above_ standard page fault exceptions (except possibly <<for loads,zcheripte_load_page_priority>>)

And

> A load page fault exception is raised when a capability load or AMO instruction is executed with C-permission granted and the virtual page’s CRG bit does not equal sstatus.CRG and the capability read from memory optionally has its tag set. **[#zcheripte_load_page_priority]#If this scheme is used the page fault is lower priority than standard page faults.#**

Reply:
I've just used xref's to the table to avoid repeating the text, have another look....